tools/pot/openvino/tools/pot/app/run.py

```diff
@@ -27,6 +27,7 @@ def main():
 
 
 def app(argv):
+    logger.warning('Post-training Optimization Tool is deprecated and will be removed in future.')
```

Comment:
We should also add deprecation warnings to the POT API.

Reply:
@daniil-lyakhov, please extend the changes accordingly.

Reply:
The problem is that such warnings will never be shown because accuracy checker tackle all POT warnings here 
https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/openvino/tools/accuracy_checker/logging.py#L98 and neither `@derecate` nor `warnings.warn` show something

So, to show warnings we should:
1) Ask user to specify `export ACCURACY_CHECKER_LOG_LEVEL=WARNING` and call AC `init_logger` function
2) Use POT logger for warnings